bin/configs/go-petstore.yaml

```diff
@@ -15,6 +15,7 @@ additionalProperties:
   packageName: petstore
   disallowAdditionalPropertiesIfNotPresent: false
   generateInterfaces: true
+  initRequiredVars: true
```

Comment:
please rename this option to "useDefaultValueInRequiredVars" instead

Reply:
addressed